cmd/neofs-cli/modules/acl/extended/create.go

```diff
@@ -67,51 +68,51 @@ func createEACL(cmd *cobra.Command, _ []string) {
 
 	containerID := cid.New()
 	if err := containerID.Parse(cidArg); err != nil {
-		cmd.Printf("invalid container ID: %v", err)
-		return
+		cmd.PrintErrf("invalid container ID: %v\n", err)
```

Comment:
in the old package style we would write as following:

```go
err := containerID.Parse(cidArg)
exitOnErr(cmd, errf("invalid container ID: %w", err))
```

are we doing to have such funcs in the new package style?

Reply:
> are we doing to have such funcs in the new package style?

Sure. But such functions should be defined in separate package to avoid dependency loop. I decided not to define such package just for this fix.